core/src/test/java/org/mskcc/oncotree/utils/OncotreeTestConfig.java

```diff
@@ -158,9 +158,6 @@ private Version setupMockVersion() {
             nextNode.setName(rawTestValueSource[pos + 1]);
             nextNode.setMainType(rawTestValueSource[pos + 2]);
             nextNode.setColor(rawTestValueSource[pos + 3]);
-            nextNode.setNci("");
```

Comment:
Just a comment for future TODO : we should add a mock crosswalk repository and add unit tests.